# TIme header of the message:
std_msgs/Header header

# Position of the UAV
float64 north
float64 east
float64 up

# Velocity of the UAV:
float64 v_north
float64 v_east
float64 v_up

# COurse from the North:
float64 course


